rac-gcp-deploy/executor/Dockerfile

```diff
@@ -1 +1,76 @@
-FROM circleci/openjdk:8-jdk
\ No newline at end of file
+ARG GCLOUD_SDK_VERSION
+FROM google/cloud-sdk:${GCLOUD_SDK_VERSION:-252.0.0-alpine}
+
+ARG SBT_VERSION
+ENV SBT_VERSION ${SBT_VERSION:-1.2.8}
+
+RUN apk --update add openjdk8
+
+RUN gcloud components install kubectl
+
+ARG HELM_VERSION
+ENV HELM_VERSION ${HELM_VERSION:-2.14.1}
+ENV HELM_BASE_URL="https://get.helm.sh"
+ENV HELM_TAR_FILE="helm-v${HELM_VERSION}-linux-amd64.tar.gz"
+
+RUN apk add --update --no-cache curl ca-certificates \
+	&& curl -L ${HELM_BASE_URL}/${HELM_TAR_FILE} |tar xvz \
+	&& mv linux-amd64/helm /usr/bin/helm \
+	&& chmod +x /usr/bin/helm \
+	&& rm -rf linux-amd64 && \
```

Comment:
The second `&&` is on the wrong line here 😄

Reply:
fixed